mtp_api/apps/account/admin.py

```diff
@@ -1,17 +1,48 @@
 from django.contrib import admin
+from django.core.urlresolvers import reverse
+from django.utils import timezone
+from django.utils.dateformat import format as format_date
+from django.utils.html import format_html_join
+from django.utils.safestring import mark_safe
+from django.utils.translation import gettext_lazy as _
 
 from account.models import Batch, Balance
+from transaction.constants import TRANSACTION_CATEGORY
 from transaction.utils import format_amount
 
 
 @admin.register(Batch)
 class BatchAdmin(admin.ModelAdmin):
     list_display = ('created', 'label', 'user', 'transaction_count')
+    exclude = ('transactions',)
+    readonly_fields = ('label', 'list_transactions', 'user')
 
     @classmethod
     def transaction_count(cls, instance):
         return instance.transactions.count()
 
+    def list_transactions(self, instance):
+        def mapper(transaction):
+            params = {
+                'amount': format_amount(transaction.amount),
+                'date': format_date(timezone.localtime(transaction.received_at), 'd/m/Y'),
+            }
+            if transaction.category == TRANSACTION_CATEGORY.CREDIT:
+                params['status'] = 'credit (%s)' % transaction.status
+            else:
+                params['status'] = 'debit'
+            return (
+                reverse('admin:transaction_transaction_change', args=(transaction.id,)),
+                '%(amount)s %(status)s, %(date)s' % params,
+            )
+
+        return format_html_join(mark_safe('<br>\n'), '<a href="{}">{}</a>', (
```

Comment:
`<br>`? 😞 What's the context of this markup? Can we wrap it in something a little more semantic (like a `ul` or paragraphs or something)?